scripts/west_commands/runners/openocd.py

```diff
@@ -388,3 +401,89 @@ def do_debugserver(self, **kwargs):
                ['-c', self.reset_halt_cmd])
         self.print_gdbserver_message()
         self.check_call(cmd)
+
+    def do_rtt(self, **kwargs):
+        pre_init_cmd = []
+        for i in self.pre_init:
+            pre_init_cmd.append("-c")
+            pre_init_cmd.append(i)
+
+        if self.thread_info_enabled and self.supports_thread_info():
+            pre_init_cmd.append("-c")
+            rtos_command = '${} configure -rtos Zephyr'.format(self.target_handle)
+            pre_init_cmd.append(rtos_command)
+
+        rtt_address = self.get_rtt_address()
+        if rtt_address is None:
+            raise ValueError("RTT Control block not be found")
+
+        rtt_cmds = [
+            '-c', 'rtt setup 0x{:x} 0x10 "SEGGER RTT"'.format(rtt_address),
+            '-c', 'rtt server start {} 0'.format(self.rtt_port),
```

Comment:
Use formatted strings

```suggestion
            '-c', f'rtt setup 0x{rtt_address:x} 0x10 "SEGGER RTT"',
            '-c', f'rtt server start {self.rtt_port} 0',
```